the docs are autogenerated by <binary name> -M, it all boils down to this source
file; I'm removing all the man errors from here
remove some manpage errors
--- a/src/generic_agent.c
+++ b/src/generic_agent.c
@@ -1480,7 +1480,7 @@
    {
    if (options[i].has_arg)
       {
-      printf("--%-12s, -%c value - %s\n",options[i].name,(char)options[i].val,hints[i]);
+      printf("--%-12s, -%c - %s\n",options[i].name,(char)options[i].val,hints[i]);
       }
    else
       {
@@ -1524,7 +1524,7 @@
    {
    if (options[i].has_arg)
       {
-      printf(".IP \"--%s, -%c\" value\n%s\n",options[i].name,(char)options[i].val,hints[i]);
+      printf(".IP \"--%s, -%c\"\n%s\n",options[i].name,(char)options[i].val,hints[i]);
       }
    else
       {
@@ -1537,10 +1537,10 @@
        ".SH INFORMATION\n");
 
 printf("\nBug reports: bug-cfengine@cfengine.org\n");
-printf(".pp\nCommunity help: help-cfengine@cfengine.org\n");
-printf(".pp\nCommunity info: http://www.cfengine.org\n");
-printf(".pp\nSupport services: http://www.cfengine.com\n");
-printf(".pp\nThis software is (C) 2008 Cfengine AS.\n");
+printf(".PP\nCommunity help: help-cfengine@cfengine.org\n");
+printf(".PP\nCommunity info: http://www.cfengine.org\n");
+printf(".PP\nSupport services: http://www.cfengine.com\n");
+printf(".PP\nThis software is (C) 2008 Cfengine AS.\n");
 }
 
 /*******************************************************************/
